shared/templates/template_common.py

```diff
@@ -206,7 +207,12 @@ def csv_map(self, filename, language):
 
             except UnknownTargetError as e:
                 if self.action == ActionType.BUILD:
-                    sys.stderr.write(str(e) + "\n")
+                    # If lang is one of these it just means that the template
+                    # doesn't implement that language. It doesn't mean that the
+                    # target is invalid.
+                    if e.lang not in ["oval", "ansible", "bash", "puppet",
```

Comment:
Are we likely to add new build targets sometime soon? It looks like these are hard coded in a bunch of places; you might want to document somewhere that this is another location that'll need to be updated if a new target is added.

Reply:
yeah, should probably be somewhere in ssgcommon.py. I will look into that.

Reply:
It's used in one other spot. I have put the list into a variable in template_common, it seems to fit well there. It's reused in generate-from-templates.py.